### PR TITLE
target usercase conditional more specifically

### PR DESCRIPTION
**Pull request by jmtroth0**
https://dimagi-dev.atlassian.net/browse/HI-147

https://github.com/dimagi/commcare-hq/pull/22276/commits/37e849956815a687e882dc46ab8eb3da2b27e9c0 @proteusvacuum is there a reason why download case history shouldn't happen for user cases? it seemed that like that was intentional in this commit so I preserved it but I wanted to confirm

@orangejenny buddy: @kaapstorm

### CODE REVIEW

File: corehq/apps/reports/templates/reports/reportdata/case_data.html [hunk 1]
```diff
@@ -206,20 +206,21 @@ <h1>{% blocktrans %}Section: {{ section_id }}{% endblocktrans %}</h1>
...
-                {% trans 'Download Case History' %}
-            </a>
-        {% if show_properties_edit %}
-            <button type="button" class="btn btn-default pull-left data-corrections-trigger">
-                <i class="fa fa-shower"></i>
-                {% trans 'Clean Case Data' %}
-            </button>
-        {% endif %}
-        {% if show_case_rebuild %}
+            {% if not usercase %}
```
Comment (orangejenny) [inline corehq/apps/reports/templates/reports/reportdata/case_data.html]: Isn't it `is_usercase`?

Comment (jmtroth0) [reply corehq/apps/reports/templates/reports/reportdata/case_data.html]: 😮

**Review by orangejenny — APPROVED** [review]

**Comment by proteusvacuum** [comment]
Hmm, I don't remember why I did that now. I think I both misunderstood that if statement (dang it django templates and your aversion to parentheses!), and had some issues with create forms in usercases that didn't play nicely with the history generation.

**Review by kaapstorm — APPROVED** [review]
I'm happy with this change. I imagine that case history for user cases could be useful, and maybe we can add a task to a backlog somewhere to make their create forms play nicer.

**Comment by jmtroth0** [comment]
Updated to show user case history as well. @proteusvacuum could you add something to a backlog at some point about the create forms?

**Review by orangejenny — APPROVED** [review]

**Pull request merged**